docs/db.md

```diff
@@ -4,21 +4,63 @@ This part describes the database schema and interactions used in the relayer.
 
 ## Schema
 
-The relayer uses a SQLite database to store and manage transaction data. The database has a single table named `transactions` with the following schema:
+The relayer uses a SQLite database to store and manage transaction data. The database has three tables:
+
+### ika_sign_requests
+
+| Column | Data Type | Description |
+|---|---|---|
+| `id` | INTEGER | The ID of the IKA sign request (primary key) |
+| `payload` | BLOB | The raw transaction data as a byte array |
+| `dwallet_id` | TEXT | The ID of the dWallet used for signing |
+| `user_sig` | TEXT | The user's signature on the transaction |
+| `final_sig` | BLOB | The final signature generated by IKA (initially NULL) |
+| `timestamp` | INTEGER | The timestamp of the request (Unix timestamp) |
+
+### ika_txs
+
+| Column | Data Type | Description |
+|---|---|---|
+| `sr_id` | INTEGER | The ID of the IKA sign request (foreign key to `ika_sign_requests`) |
+| `status` | INTEGER | The status of the IKA transaction (0: Success, 1: Failed) |
+| `ika_tx_id` | TEXT | The ID of the IKA transaction |
+| `timestamp` | INTEGER | The timestamp of the IKA transaction (Unix timestamp) |
+| `note` | TEXT |  Any notes or error messages related to the IKA transaction |
+| **`sr_id`, `ika_tx_id`** | **Primary Key** |  The combination of `sr_id` and `ika_tx_id` uniquely identifies a row in this table |
+
+### bitcoin_txs
 
 | Column | Data Type | Description |
 |---|---|---|
-| `btc_tx_id` | INTEGER | The ID of the Bitcoin transaction (primary key) |
-| `raw_tx` | BLOB | The raw transaction data as a byte array |
-| `tx_hash` | BLOB | The transaction hash as a byte array |
-| `status` | INTEGER | The status of the transaction (0: Pending, 1: Signed, 2: Broadcasted, 3: Confirmed) |
+| `sr_id` | INTEGER | The ID of the IKA sign request (foreign key to `ika_sign_requests`) |
+| `status` | INTEGER | The status of the Bitcoin transaction (0: Pending, 1: Broadcasted, 2: Confirmed) |
+| `btc_tx_id` | TEXT | The ID of the Bitcoin transaction (hash) |
+| `timestamp` | INTEGER | The timestamp of the Bitcoin transaction (Unix timestamp) |
+| `note` | TEXT | Any notes or error messages related to the Bitcoin transaction |
+| **`sr_id`, `btc_tx_id`** | **Primary Key** | The combination of `sr_id` and `btc_tx_id` uniquely identifies a row in this table |
 
 ## Functions
 
 The relayer interacts with the database using the following functions in the `dal` package:
 
-1. `InsertTx()`: Inserts a new transaction into the database.
-2. `GetTx()`: Retrieves a transaction by its ID.
-3. `GetSignedTxs()`: Retrieves all transactions with a `signed` status.
-4. `GetBroadcastedTxs()`: Retrives all transactions with a `broadcasted` status.
-5. `UpdateTxStatus()`: Updates the status of a transaction.
+### IKA Sign Requests
+
+1. `InsertIkaSignRequest(request IkaSignRequest)`: Inserts a new IKA sign request into the `ika_sign_requests` table.
```

Comment:
I  think we don't need to list all functions. Just list function "categories" (Ika, Bitcoin.... And where they are).

Reply:
And update them in change log. I mean basic function 👍🏼

Reply:
Done